test/support/UnitTest.scala

```diff
@@ -22,12 +22,12 @@
 package support
 
 import org.mockito.scalatest.MockitoSugar
-import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import org.scalatest.{ BeforeAndAfterAll, BeforeAndAfterEach }
 import org.scalatest.matchers.must.Matchers
 import org.scalatestplus.play.PlaySpec
 import play.api.mvc.Results
 
-/**
-  * This trait should be the base class for all unit tests
+/** This trait should be the base class for all unit tests
```

Comment:
Really? Even one like `DateRangeSpec`?

Reply:
I did not add this comment, however I do disagree with it as there are plenty of cases where we don't want all of those traits, like you say the date range spec.

Reply:
I know; but this is the first time I'm seeing it.

Resolving, as it's out of scope.

Reply:
I think we should delete this @rwalpole